packages/backend/src/notifications.test.ts

```diff
@@ -20,7 +21,7 @@ const keysA = {
 
 describe('Notifications', () => {
     describe('get NotificationChannels', () => {
-        it('Returns 200 and an empty array when the user has no chanels set up', async () => {
+        it('Returns 500 as global notification is turned off', async () => {
```

Comment:
I think it would be better to return an empty array here aswell, since using the BE without any notification channel is completely fine